File: 0001_0100/0085/0085.c
#include <stdio.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int n, m;

	fscanf(f, "%d%d", &n, &m);
	while (n && m)
		if (n > m)
			n %= m;
		else m %= n;
	n += m;
	while(n--)
		putc('1', q);
	return 0;
}

File: 0001_0100/0085/0085.go
package main

import (
	"fmt"
	"os"
	"strings"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var a, b int
	fmt.Fscan(f, &a, &b)
	for b > 0 {
		a, b = b, a%b
	}
	fmt.Fprint(q, strings.Repeat("1", a))
}
